README.md

```diff
@@ -18,8 +18,10 @@ We note all the input datasets are publicly available using a NASA Earthdata acc
 
 ## Installation
 
-1. `conda env update -n topsapp_env --file environment.yml` (or use [`mamba`](https://github.com/mamba-org/mamba) to speed install up)
-2. `pip install .`
+1. Clone this repo `git clone https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp.git`
+2. Navigate with your terminal to the repo.
+3. Create a new environment and install requirements using `conda env update -n topsapp_env --file environment.yml` (or use [`mamba`](https://github.com/mamba-org/mamba) to speed install up)
+4. Install the package from cloned repo using `pip install .`
```

Comment:
Two minor comments: 
* You shouldn't need the `-n topsapp_env` with `conda env update` (you would for a `conda update`) as the name is in the yaml
* typically, when cloning the repo, you'll want to do `python -m pip install -e .` -- `python -m` is the preferred invocation now for pip (the `pip` script is going away) and `-e/--editable` mean any changes you make locally will be immediately reflected in the environment